rocketpy/rocket/aero_surface.py

```diff
@@ -1889,3 +1892,180 @@ def all_info(self):
+            coefficient curve. The file must contain no headers and the first
+            column must specify the deployed level, the second column must
+            specify the Mach number and the third column must specify the drag
+            coefficient. If a Function, it must take as input the deployed
+            level and the Mach number and return the drag coefficient.
+        reference_area : int, float
+            Reference area used to calculate the drag force of the airbrakes
+            from the drag coefficient curve. Units of m^2.
+        deployed_level : float, optional
+            Current deployed level, ranging from 0 to 1. Deployed level is the
+            fraction of the total airbrake area that is deployed. Default is 0.
+        name : str, optional
+            Name of the airbrakes. Default is "Airbrakes".
+
+        Returns
+        -------
+        None
+        """
+        super().__init__(name)
+        self.cd_curve = cd_curve
+        self.cd = Function(
+            cd_curve,
+            inputs=["Deployed Level", "Mach"],
+            outputs="Cd",
+            extrapolation="zero",
+        )
+        self.reference_area = reference_area
+        self.deployed_level = deployed_level
+        self.prints = _AirbrakesPrints(self)
+        self.plots = _AirbrakesPlots(self)
+
+        # Initialize tracker lists
+        self.state_history = []
+
+    def set_deployed_level(self, deployed_level):
+        """Set airbrake deployed level.
+
+        Parameters
+        ----------
+        deployed_level : float
+            Current deployed level, ranging from 0 to 1. Deployed level is the
+            fraction of the total airbrake area that is deployed.
+
+        Returns
+        -------
+        None
+        """
+        self.deployed_level = deployed_level
+
+    def evaluate_center_of_pressure(self):
+        """Evaluates the center of pressure of the aerodynamic surface in local
+        coordinates.
+
+        For airbrakes, all components of the center of pressure position are
+        0.
+
+        Returns
+        -------
+        None
+        """
+        self.cpx = 0
+        self.cpy = 0
+        self.cpz = 0
+        self.cp = (self.cpx, self.cpy, self.cpz)
+        pass
+
+    def evaluate_lift_coefficient(self):
+        """Evaluates the lift coefficient curve of the aerodynamic surface.
+
+        For airbrakes, the current model assumes no lift is generated.
+        Therefore, the lift coefficient (C_L) and its derivative relative to the
+        angle of attack (C_L_alpha), is 0.
+
+        Returns
+        -------
+        None
+        """
+        self.clalpha = Function(
+            lambda mach: 0,
+            "Mach",
+            f"Lift coefficient derivative for {self.name}",
+        )
+        self.cl = Function(
+            lambda alpha, mach: 0,
+            ["Alpha (rad)", "Mach"],
+            "Cl",
```

Comment:
```suggestion
            "Lif Coefficient",
```

This ensures no one ever have any question about CL